'scons' cleans the environment which breaks ccache builds because
CCACHEDIR can point to an unexpected location:

| ccache arm-linux-gnueabi-gcc ... context.c
| ccache: failed to create .../serf/1.3.6-r0/.home/.ccache (No such file or directory)

Issue is described in

  http://www.scons.org/wiki/ImportingEnvironmentSettings

and because 'bitbake' cleans environment we can pass it completely
instead of trying to enumerate needed env.

Upstream-Status: Inappropriate


Index: serf-1.3.6/SConstruct
===================================================================
--- serf-1.3.6.orig/SConstruct
+++ serf-1.3.6/SConstruct
@@ -149,6 +149,7 @@ if sys.platform == 'win32':
 env = Environment(variables=opts,
                   tools=('default', 'textfile',),
                   CPPPATH=['.', ],
+                  ENV = os.environ,
                   )

 env.Append(BUILDERS = {
